<launch>

<node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1" args="load depth_image_proc/point_cloud_xyz nodelet_manager --no-bond">
    <remap from="camera_info" to="cropped_camera_info"/>
    <remap from="image_rect" to="cropped_depth_image"/>
    <remap from="depth_registered/points" to="points"/>

  </node>
</launch>
